web-app-container/Dockerfile

```diff
@@ -2,7 +2,7 @@ FROM microsoft/azure-cli:2.0.47
 
 LABEL version="1.0.0"
 LABEL maintainer="Microsoft Corporation"
-LABEL com.github.actions.name="GitHub Action for Function App on Container"
+LABEL com.github.actions.name="Deploy to Azure WebappContainer"
```

Comment:
This is not similar to function container action name.

Reply:
yes, the names are provided by PM.